src/store.js

```diff
@@ -63,7 +63,7 @@ export class Store {
     // apply plugins
     plugins.forEach(plugin => plugin(this))
 
-    if (Vue.config.devtools) {
+    if (Vue.config.devtools && options.devtools !== false) {
```

Comment:
I guess it may be cleaner to use `Vue.config.devtools` as **default value** of `options.devtools` rather than conjunction of both. like:

```js
const devtools = options.devtools !== undefined ? options.devtools : Vue.config.devtools

// ...

if (devtools) {
  // ...
}
```

Reply:
I had thought about this approach as well.  I ended up on this way because i think we want to respect someone turning it off in Vue, which acts more like a parent here.

I could see both side of it though, what do you think from that perspective?

Reply:
Ah, so if someone has this set to true in dev but ships to prod.  It would still attach to the vue devtools if __VUE_DEVTOOLS_GLOBAL_HOOK__ exists (have saw this exist before when vue devtools are off.)

Going to switch around to your recommendation.

Reply:
@ktsn updated :arrow_up: